crates/re_renderer/src/wgpu_resources/render_pipeline_pool.rs

```diff
@@ -210,13 +226,7 @@ impl GpuRenderPipelinePool {
                     re_log::info!(label = desc.label.get(), "recompiled render pipeline");
                     Some(sm)
                 }
-                Err(err) => {
-                    re_log::error!(
-                        err = re_error::format(err),
-                        "couldn't recompile render pipeline"
-                    );
-                    None
-                }
```

Comment:
Don't remove this! This literally does the logging you unnecessarily added in `create_render_pipeline` ;)